### PR TITLE
Create two different patched images one ontop of WebLogic 12.2.1.3 an…

### PR DESCRIPTION
**Pull request by mriccell**
…d one on top of 12.2.1.3 October PSU

### CODE REVIEW

**Comment by doxiao** [comment]
Do we still need the following files? I don't see any reference to them in the README.md.
1) Dockerfile
2) build.sh
3) p28076014_12213_Generic.download

**Comment by doxiao** [comment]
In both of the docker files, we need to update the following line in the beginning.

 $ sudo docker build -t oracle/weblogic:12213-update-k8s .

**Comment by doxiao** [comment]
Shall we also document how to use build.sh?

File: OracleWebLogic/samples/12213-patch-wls-for-k8s/README.md [hunk 1]
```diff
@@ -2,24 +2,24 @@ Example of Image with WLS Domain
 ================================
 There are two  Dockerfiles which extend the Oracle WebLogic install image and apply ia necesary patch for the WebLogic Kubernetes Operator 2.0. 
 
-	1)Dockerfile.patching-ontop-12213: applies two patches `p28076014` (on top of WLS 12.2.1.3) and `p27117282` (this patch is needed only if the WebLogic binary image is created manually from this Github project).
```
Comment (rosemarymarano) [inline OracleWebLogic/samples/12213-patch-wls-for-k8s/README.md]: Typos in the first sentence: install image and apply ia necesary patch -> install image and apply a necessary patch. Other than that, it looks ok to me.

**Pull request merged**